packages/web-react/src/components/Dropdown/useDropdownStyleProps.ts

```diff
@@ -1,6 +1,6 @@
 import classNames from 'classnames';
-import { useClassNamePrefix } from '../../hooks';
-import { DropdownStyleProps } from '../../types';
+import { useAlignmentClass, useClassNamePrefix } from '../../hooks';
+import { DropdownStyleProps, GridAlignmentXType, GridAlignmentYType } from '../../types';
```

Comment:
_Grid_ alignment types? Is it correct in Dropdown? 🤔

Reply:
I think this needs to be refactored.